clang/lib/CIR/CodeGen/CIRGenExpr.cpp

```diff
@@ -1376,6 +1376,34 @@ LValue CIRGenFunction::emitMaterializeTemporaryExpr(
   return makeAddrLValue(object, m->getType(), AlignmentSource::Decl);
 }
 
+LValue
+CIRGenFunction::getOrCreateOpaqueLValueMapping(const OpaqueValueExpr *e) {
+  assert(OpaqueValueMapping::shouldBindAsLValue(e));
+
+  llvm::DenseMap<const OpaqueValueExpr *, LValue>::iterator it =
+      opaqueLValues.find(e);
```

Comment:
```suggestion
  auto it = opaqueLValues.find(e);
```
Using `auto` in place of verbose iterator types improves readability.